### PR TITLE
Add documentation for built-in roles

### PR DESCRIPTION
**Pull request by eleftherias**
Ref https://github.com/stacklok/epics/issues/11

### CODE REVIEW

**Comment by eleftherias** [comment]
@teodor-yanev @jhrozek @rdimitrov If one of you has the chance to review and see if this makes sense. @JAORMX and I have some bias because we've been staring at it for so long.

File: docs/docs/how-to/add_users_to_project.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+- `permissions_manager`: Allows users to manage roles for other users within the project.
+
+## Adding a user
+To add a user to your project, follow these steps:
+
+1) Determine the User's Role: Decide the appropriate role for the user based on their responsibilities.
+
+2) Execute the Command:
+    ```bash
+    minder project role grant --sub user-id --role desired-role --project project-id
```
Comment (jhrozek) [inline docs/docs/how-to/add_users_to_project.md:28]: how would I view the roles a user has on a project>

**Review by jhrozek — DISMISSED** [review]
thank you this looks good, feel free to merge. I left two questions inline.

File: docs/docs/how-to/add_users_to_project.md (hunk 1 above)
Comment (eleftherias) [reply docs/docs/how-to/add_users_to_project.md:33]: Added it to the docs

**Review by jhrozek — APPROVED** [review]

**Pull request merged**